src/DynamoPackages/PackageManagerSearchElement.cs

```diff
@@ -36,7 +36,7 @@ public int Votes
         public string LatestVersion { get { return Header.versions != null ? Header.versions[Header.versions.Count - 1].version : String.Empty; } }
         public string LatestVersionCreated { get { return Header.versions[Header.versions.Count - 1].created; } }
 
-        public IEnumerable<string> PackageVersions { get { return Header.versions.Select(x => x.version); } }
+        public IEnumerable<string> PackageVersions { get { return Header.versions.OrderByDescending(x => x.version).Select(x => x.version); } }
```

Comment:
Do we need both `OrderByDescending(x => x.version)` and `Select(x => x.version)`? I thought only the first call is enough?

Reply:
`OrderByDescending(x => x.version) `will return the `PackageVersion `object array and is expecting an string array that's why we need to use` .Select(x => x.version)`
![image](https://github.com/DynamoDS/Dynamo/assets/61755417/475e3552-8b68-4de9-9323-92e39fd6f230)